src/templates/create-modal-block/index.vue

```diff
@@ -42,57 +42,58 @@
     {
       label: 'Domains',
       to: '/domains/create',
-      description: 'Select this card to start creating a new resource.'
+      description: 'Launch an edge application and set up security with digital certificates.'
     },
     {
       label: 'Edge Application',
       to: '/edge-applications/create',
-      description: 'Select this card to start creating a new resource.'
+      description: 'Deploy an edge application to deliver content from the edge.'
     },
     {
       label: 'Variables',
       to: '/variables/create',
-      description: 'Select this card to start creating a new resource.'
+      description: 'Create environment variables or secrets to use with configured edge functions.'
     },
     {
       label: 'Intelligent DNS',
       to: '/intelligent-dns/create',
-      description: 'Select this card to start creating a new resource.'
+      description: 'Add an authoritative DNS server for a domain by copying nameservers values.'
```

Comment:
```suggestion
      description: 'Change the authoritative DNS server for a hosted domain.'
```

@hannah-sk confere se assim fica bom! eu acho que o add/change é mais importante, pq todo domain vem com um authoritative DNS server.
a segunda parte é mais desnecessária